### PR TITLE
MSEARCH-452 - MARC authority app - Create query that returns bib/instance records linked to authority record

### PR DESCRIPTION
**Pull request by sherzodn**
## Purpose
_MARC authority app - Create query that returns bib/instance records linked to authority record_

## Approach
When I click the number of titles link on the same row of one of the Authority results 
**Then** 
The query should only return instance/bib records that are linked to that authority record UUID (also known as has $9 populated with that UUID)
AND those results should display on the Inventory App > Instance results in list
Create a new Instance search option called Authority UUID SO that When the user goes to the results list

### Changes checklist
- [ ] API paths, methods, request or response bodies changed, added, or removed
- [ ] Database schema changes
- [ ] Interface versions changes
- [ ] Interface dependencies added, or removed
- [ ] Permissions changed, added, or removed
- [ ] Check logging.

### Learning
https://issues.folio.org/browse/MSEARCH-452

### CODE REVIEW

File: src/main/resources/model/instance.json [hunk 1]
```diff
@@ -91,6 +91,11 @@
         "contributorTypeId": {
           "index": "keyword",
           "showInResponse": [ "search", "call-number-browse" ]
+        },
+        "authorityId": {
+          "index": "keyword",
+          "searchAliases": [ "authorityId" ],
+          "showInResponse": [ "search", "call-number-browse" ]
         }
```
Comment (psmagin) [inline src/main/resources/model/instance.json:99]: Please update the documentation with new search options

File: src/test/java/org/folio/search/controller/SearchInstanceIT.java [hunk 2]
```diff
@@ -64,6 +64,7 @@ void searchByInstances_parameterized_singleResult(String query, String value) th
     "uniformTitle all {value}, deja vu",
     "uniformTitle all {value}, déjà vu",
     "contributors.name all {value}, franks",
+    "contributors.authorityId == {value}, 11110000-fcf6-45cc-b6da-4420a61ef72c",
```
Comment (psmagin) [inline src/test/java/org/folio/search/controller/SearchInstanceIT.java:67]: Please add a test for the alias
`authorityId == {value}`

File: src/test/java/org/folio/search/controller/SearchInstanceIT.java [hunk 3]
```diff
@@ -249,6 +250,8 @@ private static Stream<Arguments> testDataProvider() {
       arguments("contributors.name = {value}", "Anton*"),
       arguments("contributors.name = {value}", "*rmelen, Frank"),
 
+      arguments("contributors.authorityId == {value}", "55294032-fcf6-45cc-b6da-4420a61ef72c"),
```
Comment (psmagin) [inline src/test/java/org/folio/search/controller/SearchInstanceIT.java:253]: Please add a test for the alias
`authorityId == {value}`

**Review by psmagin — CHANGES_REQUESTED** [review]
Please update the documentation with new search options

File: src/main/resources/model/instance.json (hunk 1 above)
Comment (sherzodn) [reply src/main/resources/model/instance.json:99]: done

File: src/test/java/org/folio/search/controller/SearchInstanceIT.java (hunk 3 above)
Comment (sherzodn) [reply src/test/java/org/folio/search/controller/SearchInstanceIT.java:254]: done

File: src/test/java/org/folio/search/controller/SearchInstanceIT.java (hunk 2 above)
Comment (sherzodn) [reply src/test/java/org/folio/search/controller/SearchInstanceIT.java:67]: done

**Review by psmagin — APPROVED** [review]
Karate tests?

**Review by Shans-Kaluhin — APPROVED** [review]

**Pull request merged**